### PR TITLE
v9: Bugfix - Implemented attribute changes in controller

### PR DESCRIPTION
**Pull request by Zeegaan**
Fixes https://github.com/umbraco/Umbraco-CMS/issues/10403

# Notes
- Removed attributes from UmbracoAuthorizedJsonController.cs
- Added those attributes to UmbracoAuthorizedController.cs

# How to test
- Set up a new document type with a rich text editor and tags
- Try making some content and fill out a few tags
- Upload an image in the rich text editor
- Save content
- Navigate to the contents information 
- Go to settings and click published status and try to reload the Memory Cache, this should hit all affected classes and everything should work as intended 👍

### CODE REVIEW

File: src/Umbraco.Web.BackOffice/Controllers/UmbracoAuthorizedApiController.cs [hunk 1]
```diff
@@ -23,6 +23,9 @@ namespace Umbraco.Cms.Web.BackOffice.Controllers
     /// is logged in using forms authentication which indicates the seconds remaining
     /// before their timeout expires.
     /// </remarks>
+    [AngularJsonOnlyConfiguration] // TODO: This could be applied with our Application Model conventions
+    [JsonExceptionFilter]
+    [AngularJsonOnlyConfiguration]
```
Comment (bjarnef) [inline src/Umbraco.Web.BackOffice/Controllers/UmbracoAuthorizedApiController.cs:28]: @Zeegan I guess we don't need `[AngularJsonOnlyConfiguration]` twice here.

Comment (Zeegaan) [reply src/Umbraco.Web.BackOffice/Controllers/UmbracoAuthorizedApiController.cs:28]: Nice catch 💪

Comment (Zeegaan) [inline src/Umbraco.Web.BackOffice/Controllers/UmbracoAuthorizedApiController.cs:28]:
```suggestion
```

**Review by nikolajlauridsen — APPROVED** [review]
Code looks good, going through your tests everything works fine as well, and I can confirm that the bug no longer occurs 👍.

**Pull request merged**